hack/install-pipeline-service.sh

```diff
@@ -4,20 +4,20 @@ ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/..
 
 source ${ROOT}/hack/flags.sh "The install-pipeline-service.sh installs Pipeline Service for development and testing on non-production clusters / kcp instances."
 MODE=preview parse_flags $@
-
+KCP_WORKSPACE=redhat-pipeline-service-compute
```

Comment:
let's keep the var names consistent 
```suggestion
PIPELINE_SERVICE_WORKSPACE=redhat-pipeline-service-compute
```

Reply:
Updated